modules/i18n/src/main/I18nKeys.scala

```diff
@@ -142,6 +142,7 @@ object I18nKeys:
   val `games` = I18nKey("games")
   val `forum` = I18nKey("forum")
   val `xPostedInForumY` = I18nKey("xPostedInForumY")
+  val `askConcluded` = I18nKey("askConcluded")
```

Comment:
here's more translatable strings (would need a lot more to be fully translated - if only in the help section documenting all the options)